src/Clearinghouse.sol

```diff
@@ -74,14 +78,16 @@ contract Clearinghouse is Policy, RolesConsumer, CoolerCallback {
     // --- INITIALIZATION --------------------------------------------
 
     constructor(
+        address ohm_,
         address gohm_,
         address staking_,
         address sdai_,
         address coolerFactory_,
         address kernel_
     ) Policy(Kernel(kernel_)) CoolerCallback(coolerFactory_) {
         // Store the relevant contracts.
-        gOHM = ERC20(gohm_);
+        ohm = ERC20(ohm_);
```

Comment:
You can grab the OHM address by calling IStaking(staking_).OHM();

I would move this line below line 91, after initializing staking